app/controllers/submit.js

```diff
@@ -258,6 +266,16 @@ export default Controller.extend(Analytics, BasicsValidations, COIValidations, N
     sloanDataInputEnabled: alias('features.sloanDataInput'),
     sloanPreregInputEnabled: alias('features.sloanPreregInput'),
 
+    // Variable controlled by sloan waffle flags
+    shouldShowAuthorAssertionsPanel: computed('sloanDataInputEnabled', 'sloanPreregInputEnabled', 'currentProvider.inSloanStudy', function () {
+        if (!this.get('currentProvider.inSloanStudy')) {
+            return false;
+        }
+        return this.get('sloanPreregInputEnabled') || this.get('sloanDataInputEnabled');
+    }),
```

Comment:
`shouldShowAuthorAssertionsPanel` is true if either the `sloasPreregInputEnabled` or `sloanDataInputEnabled` waffle switch is on, unless the provider opt out of the Sloan study.